domains/shibiliya.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Shibiliya's personal developer website",
+    "repo": "https://github.com/duaboola",
+    "owner": {
+        "username": "duaboola",
+        "email": "ismailshibiliya@gmail.com"
+    },
+    "record": {
+       "ALIAS": "https://duaboola.github.io/ShibiliyaIsmail.github.io/"
```

Comment:
Hi, @duaboola I don't think that this record will work correctly. Cname/Alias should be a domain or subdomain. It knows nothing about HTTP. You can either rename your GitHub username to `ShibiliyaIsmail` and use `ShibiliyaIsmail.github.io` here.